Version information is located in file: /python_packaging/gmxapi/src/gmxapi/version.py

This patch overrides the outdated data in the TOML file.
diff -ruN gromacs-v2024.1.orig/python_packaging/gmxapi/pyproject.toml gromacs-v2024.1/python_packaging/gmxapi/pyproject.toml
--- gromacs-v2024.1.orig/python_packaging/gmxapi/pyproject.toml	2024-02-28 14:31:24.000000000 +0100
+++ gromacs-v2024.1/python_packaging/gmxapi/pyproject.toml	2026-01-29 17:05:01.426362279 +0100
@@ -4,7 +4,7 @@
 
 [project]
 name = "gmxapi"
-version = "0.4.1"
+version = "0.5.0a1"
 description = "gmxapi Python interface for GROMACS."
 readme = "README.rst"
 license = { text = "GNU LESSER GENERAL PUBLIC LICENSE" }
